File: Selenium/Project/FST_HRM_Project/src/test/java/Activity/Activity_02.java
package Activity;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity_02 {

    public static void main(String[] args) {
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        driver.get("http://alchemy.hguy.co/orangehrm");
        System.out.println(driver.getTitle());
        WebElement img = driver.findElement(By.xpath("//div[@id='divLogo']/img"));
        System.out.println("Src attribute is: " + img.getAttribute("src"));
        driver.close();
    }
}

File: Python/Activities/Project/Activity_02.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from selenium.webdriver.support import color
from webdriver_manager.firefox import GeckoDriverManager
import time
service = FirefoxService(GeckoDriverManager().install())
with webdriver.Firefox(service=service) as driver:
    # Navigate to the URL
    driver.get("http://alchemy.hguy.co/orangehrm")
    print("Page Title is :", driver.title)
    driver.close()